internal/pkg/stream/ecs.go

```diff
@@ -154,6 +161,35 @@ func (s *ECSDeploymentStreamer) Fetch() (next time.Time, done bool, err error) {
 			done = true
 		}
 	}
+
+	stoppedSvcTasks, err := s.client.StoppedServiceTasks(s.cluster, s.service)
+	if err != nil {
+		if request.IsErrorThrottle(err) {
+			s.ecsRetries += 1
+			return nextFetchDate(s.clock, s.rand, s.ecsRetries), false, nil
+		}
+		return next, false, fmt.Errorf("fetch stopped tasks: %w", err)
+	}
+	s.ecsRetries = 0
```

Comment:
I'm a little confused about how `s.ecsRetries` gets used. I can only find places where its value is set, but I cannot find any of its usage 🤔  Do you happen to know? Or do you think we can remove the variable?

Reply:
we are using `s.ecsRetries` is passed as parameter to `nextFetchDate` over here.
https://github.com/aws/copilot-cli/blob/mainline/internal/pkg/stream/stream.go#L57